custom_components/tuya_local/devices/qnect_usb_powerstrip.yaml

```diff
@@ -3,6 +3,10 @@ products:
   - id: ivd9p236skiaqdrk
     manufacturer: Qnect
     model: QN-WP03
+  - id: qsjg9owomawgtoar
+    manufacturer: Gosund
+    model: PM03_T-AR (wall plug form factor)
+    model_id: 0000004535
```

Comment:
I don't think this model id is necessary, but since it is numeric, it needs double quotes to be interpreted as a string.

The model should also not have notes, the name can be overridden with "Wall plugs" instead.

My recommendation would be:
```
 - id: qsjg9owomawgtoar
    manufacturer: Gosund
    model: PM03_T-AR
    name: Wall plugs

Reply:
Your recommendation is sensible -- done!